default.nix

```diff
@@ -280,6 +280,15 @@ in
       '';
     };
 
+    enableManageSieve = mkOption {
+      type = types.bool;
+      default = false;
+      description = ''
+        Whether to enable ManageSieve, setting this option to true will open
+        port 4190 in the firewall.
```

Comment:
Maybe a quick explanation that this allows to manage sieve scripts via thunderbird etc.

Reply:
OK, done.